### PR TITLE
add zh trans content/zh/docs/reference/setup-tools/kubeadm/generated/…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+-->
+* [kubeadm join](kubeadm_join.md)	 - 在要加入现有集群的任何计算机上运行此命令
+<!--
+* [kubeadm join phase control-plane-join](kubeadm_join_phase_control-plane-join.md)	 - Join a machine as a control plane instance
+-->
+* [kubeadm join phase control-plane-join](kubeadm_join_phase_control-plane-join.md)	 - 加入机器作为控制平面实例
+<!--
+* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - Prepare the machine for serving a control plane
+-->
+* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - 准备为控制平面服务的机器
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md]:
```suggestion
* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - 准备机器用于控制平面服务
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+<!--
+* [kubeadm join phase kubelet-start](kubeadm_join_phase_kubelet-start.md)	 - Write kubelet settings, certificates and (re)start the kubelet
+-->
+* [kubeadm join phase kubelet-start](kubeadm_join_phase_kubelet-start.md)	 - 编写 kubelet 设置、证书并（重新）启动 kubelet
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md]:
```suggestion
* [kubeadm join phase kubelet-start](kubeadm_join_phase_kubelet-start.md)	 - 生成 kubelet 配置和证书并（重新）启动 kubelet
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+<!--
+* [kubeadm join phase preflight](kubeadm_join_phase_preflight.md)	 - Run join pre-flight checks
+-->
+* [kubeadm join phase preflight](kubeadm_join_phase_preflight.md)	 - 运行加入节点启动前检查
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase.md]:
```suggestion
* [kubeadm join phase preflight](kubeadm_join_phase_preflight.md)	 - 运行加入节点操作前的预备检查
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+    <tr>
+      <td colspan="2">--apiserver-advertise-address string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      If the node should host a new control plane instance, the IP address the API Server will advertise it's listening on. If not set the default network interface will be used.
+      -->
+      如果该节点托管一个新的控制平面实例，则 API 服务器将公布其正在侦听的 IP 地址。如果未设置，则使用默认网络接口。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md]:
```suggestion
      对于将要托管新的控制平面实例的节点，指定 API 服务器将公布的其正在侦听的 IP 地址。如果未设置，则使用默认网络接口。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+      -->
+      --apiserver-bind-port int32&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值：6443
+      </td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      If the node should host a new control plane instance, the port for the API Server to bind to.
+      -->
+      如果该节点托管一个新的控制平面实例，则为 API 服务器要绑定的端口。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md]:
```suggestion
      针对将要托管新的控制平面实例的节点，设置 API 服务器要绑定的端口。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+
+<!--
+SEE ALSO
+-->
+查看其他
+
+<!--
+* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - Prepare the machine for serving a control plane
+-->
+* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - 准备为控制平面服务的机器
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_control-plane-prepare_control-plane.md]:
```suggestion
* [kubeadm join phase control-plane-prepare](kubeadm_join_phase_control-plane-prepare.md)	 - 准备机器用于控制平面服务
```

**Comment by tanjunchen** [comment]
@tengqm  已经修改

**Pull request merged**